src/app/footer/footer.component.html

```diff
@@ -80,6 +80,10 @@ <h5 class="text-uppercase">Footer Content</h5>
           <a class="btn text-white"
              routerLink="info/feedback">{{ 'footer.link.feedback' | translate}}</a>
         </li>
+        <li>
+          <a class="text-white"
+             routerLink="info/accessibility">{{ 'footer.link.accessibility' | translate }}</a>
```

Comment:
I think we should move this link next to the "Cookie Settings" in order to keep the two settings together.  So, I'd propose the footer list these links in this ordering...

"Cookie settings" "Accessibility settings" "Privacy Policy" "End User Agreement", "Send Feedback"